workease.rb

```diff
@@ -224,14 +227,14 @@ def rest_timer(time, activity)
     pid = Process.fork do
       sleep time
       `paplay ./sounds/service-login.ogg`
-      `xmessage #{message} -center -timeout 3`
+      `xmessage #{message} -center -timeout 2`
```

Comment:
Why was this changed?

Reply:
3 seconds felt a little long for a message saying; 'all good, you can get back to work now'. Maybe it's just me, but when I was testing, it felt slow.